src/app/core/entity-components/entity-list/entity-list.component.ts

```diff
@@ -148,6 +148,22 @@ export class EntityListComponent<T extends Entity>
   ngAfterViewInit() {
     this.entityDataSource.sort = this.sort;
     this.entityDataSource.paginator = this.paginator;
+    // sort data according to it's label, if the data has a label
+    // (which it has when using configuration enum types)
+    // otherwise sort by default
+    this.entityDataSource.sortingDataAccessor = (
```

Comment:
@TheSlimvReal can this replace some other workaround we had setting additional fields on the entities for sorting, I think?

Reply:
Unfortunately not, the problem there was that the data was only loaded, once that row was actually rendered in the list. Therefore, for some entries the attribute was set and for others not, which resulted in wrong sorting behavior.